core/src/main/java/org/axonframework/eventhandling/AbstractEventProcessor.java

```diff
@@ -92,6 +92,11 @@ public Registration registerInterceptor(MessageHandlerInterceptor<? super EventM
         return () -> interceptors.remove(interceptor);
     }
 
+    @Override
+    public Set<MessageHandlerInterceptor<? super EventMessage<?>>> getHandlerInterceptors() {
+        return interceptors;
```

Comment:
Make sure to return a read-only version of the List (not set) here. We don't want people to think this is the method to register interceptors.